sdk/dotnet/src/ServerSentEventGenerator.fs

```diff
@@ -5,11 +5,13 @@ open StarFederation.Datastar.Utility
 /// <summary>
 /// Converts requests into SSEs, serializes, and sends to sseHandler handlers
 /// </summary>
+[<AbstractClass; Sealed>]
 type ServerSentEventGenerator =
+    static member inline private send (seeHandler: ISendServerEvent) sse = seeHandler.SendServerEvent sse
```

Comment:
This could either be a `static let` or `static member private`, but I don't think it matters much since the goal is to `inline` either way. Happy to adjust this if there's a preference.